plugin.video.kermodeandmayo/resources/lib/youtube.py

```diff
@@ -59,7 +60,7 @@ def get_search_results(query):
         
 if __name__ == "__main__":
     for playlist_id, title, thumbnail, published_at in get_playlists():
-        print
-        print title
+        print()
```

Comment:
This is likely only used for debug. We usually don't allow using print calls and force everyone to use xbmc.log. Since is the main method of a script module, I assume this is only here for debug purposes so I won't block it.